backend/canisters/community/api/src/updates/c2c_send_message.rs

```diff
@@ -13,10 +13,16 @@ pub struct Args {
     pub replies_to: Option<GroupReplyContext>,
     pub mentioned: Vec<User>,
     pub forwarding: bool,
+    #[serde(default = "bool_true")]
+    pub block_level_markdown: bool,
     pub community_rules_accepted: Option<Version>,
     pub channel_rules_accepted: Option<Version>,
     #[serde(default)]
     pub message_filter_failed: Option<u64>,
 }
 
+fn bool_true() -> bool {
```

Comment:
We can do better than this.
1. Either all messages before the blockdown feature were added are false
2. Or we detect blockklevel markdown in the message like wot the FE does